README.md

```diff
@@ -26,9 +26,9 @@ Or using our (soon to be released publically) Terraform provider
 
 ```Terraform
 resource "chainguard_account_associations" "example" {
-  group = "<< enforce group id>>"
+  group = ["<< enforce group id >>"]
```

Comment:
`group` in the provider is still a string, not a list. See https://github.com/chainguard-dev/mono/blob/eb56e6f4a9f00e2a042f25d269fbfcf0c1f0dad0/terraform-provider-chainguard/internal/provider/resource_account_association.go#L42-L49

Reply:
reverted. thanks for pointing that out